apps/src/p5lab/spritelab/commands/criterionCommands.js

```diff
@@ -506,16 +506,23 @@ export const commands = {
     return result;
   },
 
-  // Returns true text was printed this frame.
+  // Returns true if text was printed this frame.
   printedText() {
     const previousPrintLogLength = this.previous.printLogLength || 0;
-    let result = previousPrintLogLength < this.printLog.length;
+    const result = previousPrintLogLength < this.printLog.length;
+    return result;
```

Comment:
nit: could also just do `return previousPrintLogLength < this.printLog.length;`

Reply:
There's a few places like that in this file. 

**Rationale (?):** In some commands, the logic isn't as straight-forward so keeping track with a `result` variable made sense. I left in the simpler ones to make each command feel more consistent/easier to read. Thoughts?

Reply:
That seems reasonable.  I don't have a strong feeling, and if it's consistent and helps readability, seems alright.